app/presenters/tree_builder_chargeback.rb

```diff
@@ -2,7 +2,7 @@ class TreeBuilderChargeback < TreeBuilder
   private
 
   def tree_init_options
-    {:open_all => true, :full_ids => true}
+    {:open_all => true, :full_ids => true, :click_url => "/chargeback_rate/tree_select/", :onclick => "miqOnClickGeneric"}
```

Comment:
You're doing this in the wrong `TreeBuilder` as it could break stuff in the other subclasses. Can you please redefine it in the `TreeBuilderChargebackRates` and call `super.merge`? Also let's make it there a multiline and you can set `full_ids` to false (there). Maybe it would need a slight change at the `tree_select` but it should probably simplify stuff.